frontend/src/Commons/Styles/Footer.style.ts

```diff
@@ -23,6 +24,10 @@ const StyledFooter = styled.footer`
   a {
     color: ${tokens.typographyColorLink};
   }
+
+  @media (min-width: ${screenSizes.mobile}) and (max-width: 500) {
+    left: -2vh;
+  }
```

Comment:
what's up here?

Reply:
I forgot to fix that when I was going around fixing positioning but that is because when it goes higher than 500 vw it would look weird. Although hardcoded is a no, I was going to add a constant here and fix it.

Reply:
1vw = 1% of viewport width
Why do you want 5x your screens width. Also your max-width is invalid as it doesn't specify a unit.

Reply:
removed it 😛

Reply:
I was testing out things, but after many changes. I didn't need it anymore. It was supposed to be removed to after I was satisfied with the end result.